scripts/deploy.js

```diff
@@ -2,6 +2,7 @@ const fs = require("fs");
 const hre = require("hardhat");
 const { exit } = require("process");
 const { ethers } = hre;
+const namehash = require('eth-ens-namehash');
```

Comment:
True! It's already used throughout the existing test suite and it's just a script, so I'm not going to bother changing it. For something that would actually be deployed, would def want to avoid the cruft.